src/main/java/com/github/sshaddicts/lucrecium/lucreciumRunner.java

```diff
@@ -0,0 +1,174 @@
+package com.github.sshaddicts.lucrecium;
+
+import com.github.sshaddicts.lucrecium.datasets.DummyDataSet;
+import com.github.sshaddicts.lucrecium.datasets.ImageDataSet;
+import com.github.sshaddicts.lucrecium.imageProcessing.ImageProcessor;
+import com.github.sshaddicts.lucrecium.imageProcessing.Imshow;
+import com.github.sshaddicts.lucrecium.imageProcessing.Validator;
+import com.github.sshaddicts.lucrecium.neuralNetwork.RichNeuralNet;
+import com.github.sshaddicts.lucrecium.util.FileInteractions;
+import org.deeplearning4j.api.storage.StatsStorage;
+import org.deeplearning4j.nn.multilayer.MultiLayerNetwork;
+import org.deeplearning4j.ui.api.UIServer;
+import org.deeplearning4j.ui.stats.StatsListener;
+import org.deeplearning4j.ui.storage.InMemoryStatsStorage;
+import org.deeplearning4j.util.ModelSerializer;
+import org.nd4j.linalg.dataset.DataSet;
+import org.opencv.core.Core;
+
+import java.awt.*;
+import java.io.IOException;
+
+
+public class lucreciumRunner {

```

Comment:
LucreciumRunner 

btw it's a good practice to have no main classes in libraries